tests/BasicSearchTests.FunctionalTests.Core/EnvironmentSettings.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    /// <summary>
+    /// This class reads the various test run settings which are set through env variable.
+    /// </summary>
+    public class EnvironmentSettings
+    {
+        private static string _searchServiceBaseurl;
+        private static string _indexBaseUrl;
+
+        /// <summary>
+        /// The environment against which the (search service) test has to be run. The value would be picked from env variable.
+        /// If nothing is specified, int search service is used as default.
```

Comment:
Also I'm not sure if we want to have our non-prod environment urls in our repos.